tests/conftest.py

```diff
@@ -1,254 +0,0 @@
-import pytest
```

Comment:
Can you give me the high-level overview of why we no longer need this file, and what we are doing instead?